openapi.yaml

```diff
@@ -3726,6 +3741,19 @@ components:
               minItems: 1
               items:
                 type: string
+    result_status:
+      type: string
+      enum:
+        - running
+        - canceled
+        - finished
+        - error
+      description: |-
+        The status of a batch job.
+
+        This field is REQUIRED if the `partial` parameter is given.
+        This field is strongly RECOMMENDED if the job has errored.
```

Comment:
Why this recommendation note for errored jobs? 
You can only get a response with `partial` on anyway, otherwise you get `JobNotFinished` (line 3127 above)

Reply:
Also: I'm not sure "errored" is proper English (at least my spell checker complains)

Reply:
> Why this recommendation note for errored jobs? You can only get a response with `partial` on anyway, otherwise you get `JobNotFinished` (line 3127 above)

Partial results can be exposed for jobs that ~errored~ stopped due to an error. In this case ideally, I'd require the status, too. But to not introduce a breaking change, it's just recommended for now.

>  I'm not sure "errored" is proper English

Replaced it with something else.